### PR TITLE
Replace ramsey uuid package with symfony uid package and v4 to v7 Uuid

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no 
| Deprecations? | no  
| Fixed tickets | fixes # 
| Related issues/PRs | # 
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Replace ramsey uuid package with symfony uid package.

#### Why?

As we already use most symfony packages it better to build ontop of the symfony uid package instead of another third party package. This make maintainance on our side easier as we need not check anothr package for PHP compatibility or issues with the Symfony Sulu core.

### CODE REVIEW

File: src/Sulu/Bundle/SecurityBundle/EventListener/AuhenticationFailureListener.php [hunk 1]
```diff
@@ -46,10 +46,10 @@ public function onLoginFailure(AuthenticationFailureEvent $event)
 
             if ($this->passwordHasherFactory instanceof PasswordHasherFactoryInterface) {
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/SecurityBundle/EventListener/AuhenticationFailureListener.php:47]: this bridge can also be removed we should target it in an own pull request

File: packages/article/src/Domain/Model/Article.php [hunk 2]
```diff
@@ -35,7 +35,7 @@ class Article implements ArticleInterface
     public function __construct(
         ?string $uuid = null
     ) {
-        $this->uuid = $uuid ?: Uuid::uuid4()->toString();
+        $this->uuid = $uuid ?: Uuid::v7()->__toString();
```
Comment (alexander-schranz) [inline packages/article/src/Domain/Model/Article.php:38]: `->toString()` was later introduced so we need use `__toString()` for Symfony 6.4 compatibility.

Comment (alexander-schranz) [inline packages/article/src/Domain/Model/Article.php:38]: Think `v7` makes more sense for all our cases as they are time based sorted and a good replacement for v4

**Review by Prokyonn — APPROVED** [review]

**Comment by alexander-schranz** [comment]
Looks like we need replace ramsey/uuid also in the php-task. https://github.com/php-task/php-task/issues/50

/cc @mamazu if you want to have a look at this.

**Pull request merged**